pages/nft/iscn/_iscnId.vue

```diff
@@ -618,6 +619,8 @@ export default class NFTMintPage extends Vue {
       console.error(error)
       this.mintState = MintState.DONE
       this.setError(error)
+    }finally{
+      this.fetchWalletBalance()
```

Comment:
Maybe replace `getAccountBalance` call with `fetchWalletBalance` and `getBalance` from store?